src/pages/myTodo/tabs/ScrapPage.tsx

```
@@ -51,18 +46,44 @@ const ScrapPage = () => {
     );
   };
 
-  const handleRegionSelect = (regionName?: string, regionCode?: string) => {
-    console.log(regionCode);
+  const handleRegionSelect = (regionName?: string) => {
     if (regionName) {
-      if (activeTab === 'job') setJobRegion(regionName);
-      else setEduRegion(regionName);
+      activeTab === 'job' ? setJobRegion(regionName) : setEduRegion(regionName);
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)